### ISSUE TITLE
dropcap / lead-in treatment when opens with quotes

### ISSUE THREAD

**Issue by MrBryan** [TheTyee/design-article.thetyee.ca#200]
There are a number of stories that start with quotes. The drop-cap/first-character pseudoclass picks up on this and adds the size to the quote and the first letter but it looks odd. Not sure what we want to do...

![image](https://cloud.githubusercontent.com/assets/1721891/16402784/2d23111c-3cbd-11e6-9fff-20ecab634f83.png)

**Comment by alexgreen** [comment]
Having a quote mark in the dropcap isn't a big problem, although sometimes people will leave the quote small and enlarge just the letter—but enlarging only the quote mark would not be good. I'd say this is quote a minor detail that we can easily defer.

However, there are some bigger problems here:
1. The text uses a dumb quote mark, not a real quote mark. (http://webdesign.tutsplus.com/articles/start-using-quotation-marks-the-correct-way--webdesign-16905)
   I think this is very common at the Tyee and I'm not sure if the problem is that writers are using the wrong character, or that the text is getting converted improperly somewhere down the line. I think we talked about this before, but didn't decide on a solution.
   I looked into whether there's a CSS-based solution for replacing these but no dice, so I guess this may have to get replaced in the back end or via (hopefully not) javascript. Soon we'll need to start using proper quote marks as every other reputable publication does. Incorrect use of language doesn't reflect well on a newspaper's brand credibility.
2. Why is the paragraph set in italics? The only time an opening paragraph should be set in italics is in the case of an Ed's note but an ed's note should precede the drop cap, as here: http://192.168.0.60:4000/library.html

**Comment by phillipadsmith** [comment]
> I looked into whether there's a CSS-based solution for replacing these but no dice, so I guess this may have to get replaced in the back end 

This can be easily accomplished on the server / in the Bricolage template. We can just run [Smarty Pants](http://daringfireball.net/projects/smartypants/) over the text before outputting it via a `filter` in Bricolage. 

That said, I'd like to see exactely where that's happening too. @MrBryan Can you drop the URL for that story so I can take a peek?

Phillip.

**Comment by MrBryan** [comment]
The quote & use if italics would of course be editorial decisions so probably should take them up with them in case there was something intentional. It does seem to be the norm on news sites to use curly 

For the matter at hand,  the story is here:

http://preview.thetyee.ca/Opinion/2016/06/24/Vancouver-Housing-Nightmare-Long-Foretold/

I have seen times where the dropcap had inadequate padding in this situation i'll have to find those examples - I think because the font is not monospace there were odd appearances with lead "M" + quote characters which would look just as bad/worse with curly. I do think they (Michelle + Robyn)  pointed out the oddness of the quote + character as dropcap, too. I could just disable it when it starts that way if they still feel that way

**Comment by alexgreen** [comment]
> The quote & use if italics would of course be editorial decisions

...only if editorial wants to be wrong. :) There are pretty specific and universal rules of typography about which glyphs should be used where. That's why virtually every other publication conforms to the standard. The glyph you see on that Tyee page is simply not a quote mark (it is in fact a double prime which has a completely different meaning) and to use it there is indisputably wrong. It's not a matter of aesthetics. It's the same as spelling a word wrong.
The article I linked to explains this in detail.

**Comment by alexgreen** [comment]
> I have seen times where the dropcap had inadequate padding in this situation i'll have to find those examples

Yes, please link examples if see any! I haven't seen any issues yet. I can set up a test page with a lead-in paragraph for every letter of the alphabet, but it's not at the top of my list atm.

**Comment by alexgreen** [comment]
> I could just disable it when it starts that way if they still feel that way

Please don't do that. It's important to start articles consistently with a dropcap. We shouldn't depart from our typographic conventions unless we're intending to convey a specific meaning to the reader.

**Comment by MrBryan** [comment]
> **it is in fact a double prime** which has a completely different meaning) and to use it there is indisputably wrong. It's not a matter of aesthetics. It's the same as spelling a word wrong.

Don't think so, it's a straight/dumb quote (which I'm not sure everyone would quite equate as bad as a spelling mistake but worth getting right in any case) not a prime, a prime would look like this

![image](https://cloud.githubusercontent.com/assets/1721891/16568027/44049adc-41f3-11e6-90fa-9961e07be929.png)

vs what it is now:

![image](https://cloud.githubusercontent.com/assets/1721891/16568110/527635fc-41f4-11e6-8c1b-524a2e72a22b.png)

right? : https://en.wikipedia.org/wiki/Prime_(symbol)#Double_prime

But to avoid going in circles ... we all agree the norm & most correct is smart quotes so as Phillip said we'll have to figure out how the straight quotes are getting in there and fix one way or the other whether word processor / defaults or editor habits. I'll post a thread on Basecamp about it so that editors can weigh in where they think it comes from. 

Let's consider this closed unless we find an example of a dropcap that has odd padding or editorial brings up any issue with dropcap+ quotes in reviewing pages - it was only casually mentioned to me as looking odd when I was in the office.

**Comment by phillipadsmith** [comment]
> I'll post a thread on Basecamp about it so that editors can weigh in where they think it comes from.

Super. And I can sit down with Robyn for 5 mins to resolve after you've posted.

**Comment by phillipadsmith** [comment]
Another example, http://preview.thetyee.ca/Opinion/2016/03/08/BC-Carbon-Tax-Failure/

**Comment by phillipadsmith** [comment]
Another drop cap edge case:

Drop cap splits word at beginning of story
URL: http://preview.thetyee.ca/News/2016/05/25/Combat-Training-Natural-Resource/
Description: This story starts with ‘B.C.’ The format is just grabbing the ‘B.’ and making that the drop cap, while the ‘C.’ is in regular body text.
Reporter: Jeanette

<img width="443" alt="tyee_staff_consolidated_article_page_feedback_-_google_docs" src="https://cloud.githubusercontent.com/assets/34128/16674803/adba4fec-4470-11e6-8e72-5594e1229e42.png">
